src/main/java/seedu/address/model/client/insurance/claim/Claim.java

```diff
@@ -14,7 +14,7 @@ public class Claim {
     public static final String INVALID_CLAIM_AMOUNT = "Claim Amount must be a positive integer";
 
     private final String claimId;
-    private final boolean isOpen;
```

Comment:
Made non final so that we are able to modify this state to close the claim.

Reply:
noted